docs/user-guide/configure-origins.md

```diff
@@ -0,0 +1,180 @@
+# Configuring Backend Services and Origins
+
+This document describes 
+## Configuration file
+
+Backend services and origins known to Styx are configured in their own YAML file, which is referenced by the main 
+configuration via the `originsFile` property in the services block.
+
+The value for this `originsFile` property accepts including environment variables (with the ${ENV_VAR} format) and can reference:
+ - A path in the filesystem by default.
+ - A resource in the classpath by using the prefix *classpath:*.
+
+###Examples:
```

Comment:
This heading doesn't render correctly. I believe you need to add a whitespace character between the ### marks and the heading text.